woocommerce/wc-admin
# Remove onboarding feature flag and a/b testing

Fixes #4575 

This PR removes the feature flag and permanently enables the onboarding feature since the old onboarding is being removed from core.

Note: I had a little trouble with the autoloader in core and was not able to load the plugin when checking out `fix/26173` in core.  Running `composer dump-autoload` seemed to fix things, but not sure if something else has changed here or just an issue with my local env. /cc @rrennick 

### Detailed test instructions:

1. Check out the WooCommerce branch from this PR - https://github.com/woocommerce/woocommerce/pull/26853
2. Check that no console/network errors occur during onboarding (specifically after the store details step)
3. Check on a new site that store pages are still created.
4. Check the “Help” tab under any core WooCommerce related page and make sure the “Setup wizard” tab still exists and works as expected.
5. Checkout an older version of WooCommerce and make sure help tabs are correctly overridden.

### Changelog Note:

Dev: Remove onboarding split testing.
